ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -18,6 +19,8 @@ struct gcp_file_handle;
 struct gcp_store {
     WT_STORAGE_SOURCE storage_source;
     WT_EXTENSION_API *wt_api;
+    std::shared_ptr<gcp_log_system> log;
```

Comment:
Why is this a `shared_ptr`? It looks like `unique_ptr` would be more appropriate for the use case.

Reply:
The reason I used a shared_ptr here is because AddBackend() which is a function used by logSink to add in the log system expecting a `shared_ptr` to the log system to be passed in as an argument.

Reply:
Gotcha, fair enough then.